Aptoide/appcoins-wallet-android
# [APPC-1059] Remove network managing reference

What does this PR do?

This PR removes all the networks that are not the main ethereum network or the ropsten network. 
This also removes the possibility of the user changing networks on the wallet. 
The networks are now set on default depending on the type of build the application is running

This ticket also changes the version name and icon of the dev version
Where should the reviewer start?

Web3jProvider.java
EthereumNetworkRepository.java (deleted)
SettingsFragment.java

How should this be manually tested?

Check if the dev version is using the ropsten network and the prod is using the main ethereum network
Do a transaction and check if it’s done 
The option in settings to change network should be gone
The icon from the dev version should have dev written on it
In settings the version name should now have .dev in front of the version 

Tickets related

https://aptoide.atlassian.net/browse/APPC-1059
